src/NetAppFiles/NetAppFiles/Pool/UpdateNetAppFilesPool.cs

```diff
@@ -0,0 +1,133 @@
+using Microsoft.Azure.Commands.NetAppFiles.Common;
+using Microsoft.Azure.Commands.NetAppFiles.Models;
+using Microsoft.Azure.Management.NetApp;
+using Microsoft.Azure.Management.NetApp.Models;
+
+namespace Microsoft.Azure.Commands.NetAppFiles.Pool
+{
+    [Cmdlet(
+        "Update",
+        ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "NetAppFilesPool",
+        SupportsShouldProcess = true,
+        DefaultParameterSetName = FieldsParameterSet), OutputType(typeof(PSNetAppFilesPool))]
+    [Alias("Update-AnfPool")]
+    public class UpdateAzureRmNetAppFilesPool : AzureNetAppFilesCmdletBase
+    {
+        [Parameter(
+            Mandatory = true,
+            ParameterSetName = FieldsParameterSet,
+            HelpMessage = "The resource group of the ANF account")]
+        [ValidateNotNullOrEmpty]
+        [ResourceGroupCompleter()]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ParameterSetName = FieldsParameterSet,
+            HelpMessage = "The location of the resource")]
+        [ValidateNotNullOrEmpty]
+        [LocationCompleter("Microsoft.NetApp/netAppAccounts/capacityPools")]
+        public string Location { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ParameterSetName = FieldsParameterSet,
+            HelpMessage = "The name of the ANF account")]
+        [ValidateNotNullOrEmpty]
+        [ResourceNameCompleter(
+            "Microsoft.NetApp/netAppAccounts",
+            nameof(ResourceGroupName))]
+        public string AccountName { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ParameterSetName = FieldsParameterSet,
+            HelpMessage = "The name of the ANF pool")]
+        [Parameter(
+            Mandatory = true,
+            ParameterSetName = ParentObjectParameterSet,
+            HelpMessage = "The name of the ANF pool")]
+        [ValidateNotNullOrEmpty]
+        [Alias("PoolName")]
+        [ResourceNameCompleter(
+            "Microsoft.NetApp/netAppAccounts/capacityPools",
+            nameof(ResourceGroupName),
+            nameof(AccountName))]
+        public string Name { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "The size of the ANF pool")]
+        [ValidateNotNullOrEmpty]
+        public long? PoolSize { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "The service level of the ANF pool")]
+        [ValidateNotNullOrEmpty]
+        public string ServiceLevel { get; set; }
+
+        [Parameter(
+            ParameterSetName = ParentObjectParameterSet,
+            Mandatory = false,
+            ValueFromPipeline = true,
+            HelpMessage = "The account object containing the pool to update")]
+        [ValidateNotNullOrEmpty]
+        public PSNetAppFilesAccount AccountInputObject { get; set; }
```

Comment:
@leonardbf nit: I would rename this to just `AccountObject`

Reply:
Ok. Renamed.